package/CHANGELOG

```diff
@@ -22,6 +22,10 @@ The rules for this file:
  * 2.8.0
 
 Fixes
+ * Fix failure in double-serialization of TextIOPicklable file reader.
+   (Issue #3723, PR #3722)
+ * Fix failure to preserve modification of coordinates after serialization.
```

Comment:
There's a change in behaviour here where pickling then unpickling no longer resets a frame to the data held in that frame.  E.g. if I load frame 10, then unwrap these coordinates (modifying them from the file held state), then serialise/deserialise that (e.g. in a multiprocessing situation) the frame is now unwrapped on deserialisation.  This is probably a good thing (tm).  This should probably get explained in an issue and/or changelog entry

Reply:
Thanks! I have added an issue regarding it https://github.com/MDAnalysis/mdanalysis/issues/4633